lib/sqli/sqli_parser.y

```diff
@@ -99,7 +99,7 @@ data_cont:
         | after_exp_cont
         ;
 
-update: TOK_UPDATE[tk1] colref_exact TOK_SET[tk2] expr_list {
+update: TOK_UPDATE[tk1] tableref TOK_SET[tk2] expr_list {
```

Comment:
According to https://docs.microsoft.com/ru-ru/sql/t-sql/queries/update-transact-sql?view=sql-server-2017 there is also a three-dots syntax:
```<object> ::=  
{   
    [ server_name . database_name . schema_name .   
    | database_name .[ schema_name ] .   
    | schema_name .  
    ]  
    table_or_view_name}
```
Moreover, how would look column references to such a table without an alias? Is it possible to use four dots style like below?
`server_name . database_name . schema_name . table_name . column_name`

In any case, what about to not have a special syntax for tableref but instead just to extend the colref_exact syntax? If you believe that a special syntax for tableref is necessary then I think we should name it as `tableref_exact`, not as `tableref`, to not mix it with existant `table_ref`.